ocs_ci/deployment/deployment.py

```diff
@@ -243,6 +308,13 @@ def subscribe_ocs(self):
         channel = config.DEPLOYMENT.get('ocs_csv_channel')
         if channel:
             subscription_yaml_data['spec']['channel'] = channel
+        if (
+            config.DEPLOYMENT.get('stage')
+            and config.DEPLOYMENT.get('stage_namespace')
```

Comment:
you can save the stage_namespace to some variable above line 311 and use it on both places instead of call get method twice

Reply:
Since the setting of `stage_namespace` is also mandatory (see previous comment), I've removed it from the condition and used it without the `get(...)` method. Fixed by 6ab36d8.